src/middleware/i18n.js

```diff
@@ -24,7 +24,7 @@ function setup() {
    * @returns {void}
    */
   return wrap(function* middleware(req, res, next) {
-    if (res.body) {
+    if (res.body && i18n) {
```

Comment:
Do we really want to allow i18n to be undefined? This may lead to undefined behavior or crashes if we reply invalid data to clients

Reply:
undefined is falsey, what is your point ?